pmm-app/src/pmm-qan/panel/components/Overview/components/QanTable/Table.tsx

```diff
@@ -209,15 +205,16 @@ export const Table: FC<TableProps> = ({
               <Spinner />
             </div>
           ) : null}
-          {!rows.length && !loading ? (
-            <div data-qa="table-no-data" className={styles.empty(scroll.y)}>
-              {noData || <h1>No data</h1>}
-            </div>
-          ) : null}
-          {rows.length && !loading ? (
+          {!loading ? (
             <div {...getTableProps()} className="table">
               <div {...getTableBodyProps()} className={cx('table-body', styles.tableBody(scroll.y))}>
-                {rows.map(RenderRow)}
+                {headerGroups.map(RenderHeader)}
+                {!!rows.length && rows.map(RenderRow)}
+                {!rows.length && (
+                  <div data-qa="table-no-data" className={styles.empty(scroll.y)}>
+                    {noData || <h1>No data</h1>}
```

Comment:
Shouldn't there be a `??` instead of `||`?

Reply:
`||` is right, if `noData` variable is empty it shows the `h1`

Reply:
Provided `noData` is properly typed, we are quite safe to use `||` as well.